sdks/python/apache_beam/tests/pipeline_verifiers.py

```diff
@@ -92,13 +93,21 @@ def _read_with_retry(self):
     matched_path = ChannelFactory.glob(self.file_path)
     if not matched_path:
       raise IOError('No such file or directory: %s' % self.file_path)
+
+    logging.info('Find %d files in %s: \n%s',
+                 len(matched_path), self.file_path, '\n'.join(matched_path))
     for path in matched_path:
       with ChannelFactory.open(path, 'r') as f:
         for line in f:
           read_lines.append(line)
     return read_lines
 
   def _matches(self, _):
+    # Wait to have output file ready on FS
+    wait_time = 20
```

Comment:
Can yo make this configurable, I would assume that not every FS will need such a wait.